src/base/QXmppJingleIq.cpp

```diff
@@ -922,9 +980,8 @@ void QXmppJingleIq::toXmlElementFromChild(QXmlStreamWriter *writer) const
     }
     d->reason.toXml(writer);
 
-    // ringing
-    if (d->ringing) {
-        writer->writeStartElement(QStringLiteral("ringing"));
+    if (const auto rtpSessionState = d->rtpSessionState; rtpSessionState != QXmppJingleIq::None) {
+        writer->writeStartElement(JINGLE_RTP_SESSION_STATES.at(rtpSessionState));
         writer->writeDefaultNamespace(ns_jingle_rtp_info);
         writer->writeEndElement();
```

Comment:
why are you copying into `const auto rtpSessionState` here? you can even have a simpler if statement if you don't do it